concrete/src/config.rs

```diff
@@ -212,11 +210,11 @@ impl ConfigBuilder {
         self
     }
 
-    #[cfg(feature = "integers")]
-    pub fn enable_custom_uint8(mut self, parameters: FheUint2Parameters) -> Self {
-        self.config.integer_config.uint8_params = Some(parameters);
-        self
-    }
+    // #[cfg(feature = "integers")]
+    // pub fn enable_custom_uint8(mut self, parameters: FheUint2Parameters) -> Self {
+    //     self.config.integer_config.uint8_params = Some(parameters);
+    //     self
+    // }
```

Comment:
I commented it out, as I wanted to find a nice way to let users customize the static `FHeUint{8, 12, 16}` types
which checks on the validity of parmeters (mainly making sure the users gives parameters that represents 8 bits for FheUint8) being done at _compile time_ and not runtime. 

I had an solution but the compile errors were not that easy to understand I think. And since we will add
more representation, It would make this part harder.

To **TDLR** for now I'm just removing the ability to customize these parameters and will remove these commented out lines.